net/aria2/Makefile

```diff
@@ -52,19 +54,19 @@ endef
 CONFIGURE_ARGS += \
 	--disable-nls \
 	$(if $(CONFIG_ARIA2_NOSSL),--disable,--enable)-ssl \
+	$(if $(CONFIG_ARIA2_BITTORRENT),--enable,--disable)-bittorrent \
+        $(if $(CONFIG_ARIA2_METALINK),--enable,--disable)-metalink \
```

Comment:
minor whitespace error. 
(other lines have properly tabs, while this has spaces?)